### PR TITLE
Updated dependencies and test scripts

### PR DESCRIPTION
**Pull request by joachimmetz**
* updated dependencies.ini and test_dependencies.ini 
* added Travis CI configuration for testing with Ubuntu 18.04 GIFT PPA dev track builds
* added Travis CI configuration for testing Python 3.6, 3.7, 3.8 with tox
* update setup.py to support Python 3.6 and later since 3.5 is not supported by sigmatools
* changed MANIFEST.in to include test_requirements.txt in sdist used by tox

### CODE REVIEW

**Comment by joachimmetz** [comment]
@berggren updates made, had to update some packages in GIFT PPA dev as well, existing and additional tests are now passing, so PR ready for review. Let me know if there are questions. IMHO next steps are to add a Dockerfile build test.

File: config/travis/install.sh [hunk 1]
```diff
@@ -32,36 +43,47 @@ then
...
 	docker cp ../timesketch ${CONTAINER_NAME}:/
 
-	if test ${TARGET} = "pypi";
+	if test "${TARGET}" = "pypi";
 	then
 		docker exec ${CONTAINER_NAME} sh -c "cd timesketch && pip3 install -r requirements.txt";
 		docker exec ${CONTAINER_NAME} sh -c "cd timesketch && pip3 install -r test_requirements.txt";
-		docker exec ${CONTAINER_NAME} sh -c "(cd timesketch/api_client/python && python3 setup.py build && python3 setup.py install)";
-		docker exec ${CONTAINER_NAME} sh -c "(cd timesketch/importer_client/python && python3 setup.py build && python3 setup.py install)";
+		# docker exec ${CONTAINER_NAME} sh -c "(cd timesketch/api_client/python && python3 setup.py build && python3 setup.py install)";
```
Comment (berggren) [inline config/travis/install.sh:84]: Can you add a comment to explain the reason why these are disabled?

File: config/travis/run_python3.sh [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
 
 python3 ./run_tests.py
 
 python3 ./setup.py build
+
 python3 ./setup.py sdist
+
 python3 ./setup.py bdist
+
+# python3 ./setup.py install
```
Comment (berggren) [inline config/travis/run_python3.sh:19]: Can you add a comment to explain the reason why this one is disabled?

File: config/travis/install.sh (hunk 1 above)
Comment (joachimmetz) [reply config/travis/install.sh:84]: They don't appear to be needed I can remove them as well

File: config/travis/run_python3.sh (hunk 2 above)
Comment (joachimmetz) [reply config/travis/run_python3.sh:19]: Let me test enabling this.

**Pull request merged**